src/axom/primal/geometry/Tetrahedron.hpp

```diff
@@ -220,6 +220,23 @@ class Tetrahedron
     return res;
   }
 
+  /*! 
+   * \brief Returns whether point \a p is contained within the tetrahedron (within tolerance \a eps)
+   */
+  bool contains(const PointType& p, double eps = 1e-8) const
+  {
+    if(axom::utilities::isNearlyEqual(ppedVolume(), 0., eps))
+    {
+      return false;
+    }
+
+    const auto bC = physToBarycentric(p);
+
+    return (bC[0] >= (0.0 - eps)) && (bC[1] >= (0.0 - eps)) && (bC[2] >= (0.0 - eps)) &&
```

Comment:
Is there a reason not to use `-eps` instead of `(0.0 - eps)`?

Reply:
I guess it makes it clear we are testing each coordinate between 0 and 1.

Reply:
I think I had that there originally for symmetry w/ the `1.0` case